interfaces/builtin/basedeclaration.go

```diff
@@ -355,6 +355,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  realsense:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
Here I follow similar test scheme in home_test.go and lxd_support_test.go.

Now that this interfaces is declared deny-auto-connection in basedeclaration. So maybe I could remove the TestAutoConnect case?